lib/smart_proxy_ansible/runner/ansible_runner.rb

```diff
@@ -174,7 +174,7 @@ def tags_cmd
       end
 
       def check_cmd
-        check_mode? ? '--check' : ''
+        check_mode? ? '"--check"' : ''
```

Comment:
This is not correct, I believe that if we use both tags and checkmode, it will not work as expected, if we need the extra `"` we should wrap the whole `cmdline`, right?

Reply:
I was a bit skeptical about that as well, but it seems to be working when using check mode and tags together.

Reply:
From what I saw it does not work any other way. I think the reason for that is the --cmdline is an option of ansible-runner but the --check is not. try to run it without and you will see.

Reply:
I'm not disputing the need of the quotes, I was just saying it should be one argument with the tags together, but if Ondra says it works, lets get it in :)